pattern-library/sass/patterns/_buttons.scss

```diff
@@ -120,8 +120,8 @@ $btn-small-font-size: font-size(small);
     &.is-hovered,
     &:focus,
     &.is-focused {
-        border-color: palette(primary, accent);
-        background: palette(primary, accent);
+        border-color: $btn-focus-color;
+        background: $btn-focus-color;
         color: palette(primary, x-back);
```

Comment:
I'd understand if you see this as out of scope, but would you mind making this a variable too?